src/chi/components/carousel/carousel.scss

```diff
@@ -0,0 +1,77 @@
+@import '_global-variables';
+@import '_global-mixins';
+
+.chi-carousel {
+  position: relative;
+  width: 100%;
+  z-index: 0;
+
+  &__content {
+    overflow: hidden;
+    width: 100%;
+
+    .chi-carousel__wrapper {
+      transition-property: transform;
+      transition-duration: 0.35s;
+      width: fit-content;
+    }
+  }
+
+  .chi-carousel__control {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    width: 20%;
+    z-index: 10;
+
+    &.-previous,
+    &.-next {
+      chi-button {
+        button {
+          &.chi-button {
+            &.-float {
+              border: 0;
+            }
+          }
+        }
+      }
+    }
+
+    &.-previous {
+      left: 0;
+      justify-content: start;
+    }
+
+    &.-next {
+      right: 0;
+      justify-content: end;
+    }
+  }
+
+  &__dots {
+    display: flex;
+    justify-content: center;
+
+    .chi-carousel__dot {
```

Comment:
Can we change the dot size to $icon-size-xs or create a $carousel-dot-size variable for it and make $icon-size-xs the value for border-radius, height, and width?

We'll also need to adjust the dot color to stay consistent with lumen.com. The non-active dots [they're using](https://www.lumen.com/en-us/about/4th-industrial-revolution.html) have a transparent background-color with a border: 0.0625rem solid $color-border-primary.

![image](https://user-images.githubusercontent.com/130654/146101189-bc904ab0-44fd-4c5c-9233-44fd0c36ed25.png)

Reply:
They are also using a `transition: all 0.2s ease-in;` on the dots which adds a subtle but nice animation

Reply:
They don't have a hover state but we could change the value of $carousel-dot-hover-background-color to $color-background-primary